doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -1,5 +1,96 @@
+
+- Identify in detail the requirements of test-cases (mapped from RA2 and RM)
+- Test criteria that shows a certain capability or feature of the system-under-test exists and behaves as expected
+- An E2E framework for conformance of Kubernetes infrastructures and workloads, including specification for conformance test infrastructure (lab environment and tools)
+- Analysis to identify where the gaps are in the industry for implementing conformance test objectives (tooling, methods, process, etc)
+
+**Not in Scope**
+- Functional testing / validation of the application provided by the workload is outside the scope of this work
+- Testing to confirm specific versions of APIs is out of scope
+- VNFM/NFVO, like ONAP, is not used in the process flow for infrastructure verifications or validations
+- Upgrades to workloads, and the respective processes of verifying upgrade procedures and validating (testing) the success and compatibility of upgrades is not in scope
+
+<a name="1.4"></a>
+## 1.4 Guidelines
+
+The objectives of the Reference Conformance for cloud infrastrucute is to verify implementations against the reference architecture which satisfies infrastructure needs for workloads. The objectives of the Reference Conformance for workloads is to verify workload implementations consume resources and behave as expected against the reference architecture.
+
+These guidlines will drive RC2 deliverables:
+
+- RC2 requirements are completely derived from RM and RA2 which specify infrastructure capabilities including compute, memory, storage, resource capabilities, performance optimization capabilities, and monitoring capabilities.
+
+- Requirements in the RM and RAs that are performance related may not have minimum perfromance criteria identified but where feasible will have tests with metrics to show relevant capabilities are present and working as expected.
+
+- Must/shall conformance criteria are testable as pass/fail and ensure requirements are met to ensure minimum thresholds of functional operability and/or performance behavior. This is the focus of RC2 since this is what will drive a commercially significant badging program.
+
+- Should/may conformance criteria, which may or may not be testable, provide recommendations or best-practices for functional operability and/or performance behavior. These criteria and associated tests can be very useful for developing, evaluating or deploying a cloud infrastructure but are not critical to a commercially significant badging program.
+
+<a name="1.5"></a>
+## 1.5 Verification Methodologies
+
+The RC2 test suit will provide verification to ensure workloads can interoperate with the CNTT RA2 infrastructure. Upstream projects will define features/capabilities, test scenarios, and test cases to be executed. 3rd Party test platforms may also be leveraged, if desired.
```

Comment:
According to the definitions above, verification is based on reviews and validation is based on runtime testing.

```suggestion
The RC2 test suite will provide validation to ensure workloads can interoperate with the CNTT RA2 infrastructure. Upstream projects will define features/capabilities, test scenarios, and test cases to be executed. 3rd Party test platforms may also be leveraged, if desired.
```